src/containers/flows/ClaimNFT/index.tsx

```diff
@@ -79,12 +81,12 @@ const ClaimNFTContainer = ({
 			return;
 		}
 
-		if (dropStage === Stage.Whitelist && !countdownDate) {
-			window?.open(
-				'https://zine.wilderworld.com/moto-genesis-nft-rewards/',
-				'_blank',
-			);
-		}
+		// if (dropStage === Stage.Whitelist && !countdownDate) {
```

Comment:
This condition wouldn't be used for future claim drops, it was previously raised as an issue in feature review. Maybe good to remove the commented code.